### DISCUSSION TITLE
New version of RSS plugin for Typemill V2

### DISCUSSION THREAD

**Discussion by gchiesa** [typemill/typemill#468] [Show and tell]
Hello, I've published a new version of RSS plugin for Typemill V2. 
It implements same functionalities as previous one, with some bug addressed in the meanwhile. 

The source and package is release in my github: https://github.com/gchiesa/typemill-plugin-rss 

Demo available on my blog: https://g-c.dev/rss

Feel free to make it available in the [plugin directory](https://plugins.typemill.net/)

**Comment by trendschau** [comment]
Great, I published it and it will also go into the newsletter today or tomorrow!
